### DISCUSSION TITLE
Custom tags to markup

### DISCUSSION THREAD

**Discussion by alexekdahl** [winkjs/wink-nlp#47] [Help on coding] [answered]
Is it possible to add a custom tag to the markup function? If not, how do I accomplish that?

Big love

**Comment by sanjayaksaxena** [comment, marked as answer]
Hi @AlexEkdahl 

Yes, it is possible to use other tags (i.e. other than `mark` tag). You can look at the first four Observable notebooks in the [WinkNLP Recipes collection](https://observablehq.com/collection/@winkjs/winknlp-recipes) — they use custom tags to create different visualizations.
